libraries/wallet/wallet.cpp

```diff
@@ -447,8 +446,11 @@ namespace golos { namespace wallet {
                         trx.operations.begin(), trx.operations.end(), std::back_inserter(op.proposed_operations),
                         [](const operation& op) -> operation_wrapper { return op; });
 
-                    op.review_period_time = time_converter(review_period_time,
+                    auto review_period_time_sec = time_converter(review_period_time,
                         dyn_props.time, dyn_props.time + (STEEMIT_MAX_PROPOSAL_LIFETIME_SEC / 2)).time();
+                    if (review_period_time_sec > time_point_sec::min()) {
```

Comment:
If `review_period_time`  is empty , the result will be `dyn_props.time`, when checking is done on `time_point_sec::min()`